build/beforeBuildSave/README.md

```diff
@@ -1,7 +1,13 @@
 Artifactory Before Build Save User Plugin
 =========================================
 
-*This plugin is currently being tested for Artifactory 5.x releases.*
-
 Whenever a build is submitted, this plugin removes any parameters from the MIME
-types of any of the artifacts in the build.
\ No newline at end of file
+types of any of the artifacts in the build.
+
+e.g The build.json file contains the information type "war;x=10".
+ 
+When uploaded the type will be "war"
+
+***Installation***
+
+Place plugin under <Artifactory_Home>/etc/plugins/
```

Comment:
@AngelloMaggio the README file should have at least the following content:

- Description
- Features
- Installation
- Usage

You can use [this README.md file](https://github.com/JFrogDev/artifactory-user-plugins/tree/master/checksums) as a reference.

Reply:
@elioengcomp I see all of that information in this file already. Do you not?

Reply:
The information is there, but I think it would be better to have the same topics structure in all plugin pages. We should start using some pattern to make it easier to the customers to use this documentation.

Reply:
@elioengcomp Makes sense. Although I doubt the pattern would make anything easier if you can fit the documentation in a tweet anyway, but I guess there probably isn't any harm in doing it regardless. I'd say maybe omit the Features section, since it doesn't really make sense in this case.